data/teams.csv

```diff
@@ -1,4 +1,4 @@
-team_id,franchiseId,teamName,abbreviation,Stadium,link

```

Comment:
had to change headers in teams.csv to follow better ruby syntax

Reply:
horrible syntax. NOT ON MY WATCH, aurora, megan, and tim!